src/top-bar/org-unit-select/org-unit-select.js

```diff
@@ -20,19 +22,55 @@ const OrgUnitSelect = () => {
         openedSelect,
         setOpenedSelect,
     } = useSelectionContext()
+    const { searchText, orgUnits, loading, setSearchText } = useOrgUnitSearch();
```

Comment:
I would prefer this to be its own PR. As well as the search in workflows. I think that is scope-creep, and this should only include the catcombo logic...